docs/essentials/building-monorepo-projects.md

```diff
@@ -221,6 +221,47 @@ promotions:
       when: "branch = 'master' and result = 'passed' and change_in('/docs/')"
 ```
 
+## Additional examples of monorepo configuration
+
+### When a directory changes
+
+```yaml
+blocks:
+  - name: Test WEB server
+    run:
+      when: "change_in('/web-app/')"
+```
+
+### When a file changes
+
+```yaml
+blocks:
+  - name: Unit tests
+    run:
+      when: "change_in('../Gemfile.lock/')"
```

Comment:
I just noticed this, it should be:
```
when: "change_in('../Gemfile.lock')"
``` 
notice that there is no `/` at the end since `Gemfile.lock` is a file and not a directory.
Also needs to be changed in the other document.

Reply:
@DamjanBecirovic I tested this last week in my project and it worked with the file with the `/` at the end. I'll update the docs anyway.

Reply:
In that case, it is more of a bug than a feature. Thanks for updating the docs :+1: